08LessonEight/in-class-project/css/style.css

```diff
@@ -41,11 +41,11 @@ footer {
 }
 
 aside h2 {
-  font-size: 14px;
+  font-size: 10px;
 }
 
 aside p {
-  font-size: 12px;
+  font-size: 10px;
 }
 
 article {
```

Comment:
Great job on the readability in the CSS file. This needs to translate over to the HTML side.